libnavui-voice/src/main/java/com/mapbox/navigation/ui/voice/options/MapboxSpeechApiOptions.kt

```diff
@@ -4,16 +4,19 @@ package com.mapbox.navigation.ui.voice.options
  * MapboxSpeechApiOptions.
  *
  * @param baseUri base URL
+ * @param gender voice gender
  */
 class MapboxSpeechApiOptions private constructor(
-    val baseUri: String
+    val baseUri: String,
+    @VoiceGender.Type val gender: String?
```

Comment:
There's a bit of inconsistency here.

`language` option is in `MapboxSpeechApi.language`:
https://github.com/mapbox/mapbox-navigation-android/blob/96c1aae72777f5b0e783ab32b0b77d918c5ba92a/libnavui-voice/src/main/java/com/mapbox/navigation/ui/voice/api/MapboxSpeechApi.kt#L32

But `MapboxSpeechApi` also accepts `MapboxSpeechApiOptions` and it looks like that would be better to keep all the parameters in `MapboxSpeechApiOptions` (including language and gender), which we can't do because of breaking changes.

Despite inconsistency we have now, I decided to add new parameter in `MapboxSpeechApiOptions`, and in case of need, all the new parameters should also be added to `MapboxSpeechApiOptions`.

@VysotskiVadim @dzinad what do you think?